presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java

```diff
@@ -988,7 +995,7 @@ public static String getPrefilledColumnValue(HiveColumnHandle columnHandle, Hive
             return String.valueOf(fileSize);
         }
         if (isFileModifiedTimeColumnHandle(columnHandle)) {
-            return String.valueOf(fileModifiedTime);
+            return HIVE_TIMESTAMP_PARSER.withZone(hiveStorageTimeZone).print(fileModifiedTime);
```

Comment:
@findepi, I removed the fixed zone in UTC time, but had add a change to pass the hive storage zone through to here. Otherwise the time was printed in UTC but was tagged with the hiveStorageTimeZone.

Reply:
What is the value being formatted here?
sth like 
```
2020-05-05T21:15:22.579
```
or with zone
```
2020-05-05T19:15:22.595Z
```
?

If you format with zone, you shouldn't need `hiveStorageTimeZone` param

can you try this?

Reply:
The Hive timestamp format is like the first one, without a zone. https://github.com/prestosql/presto/blob/master/presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java#L205

The page source is hardcoded to use this parser (without zone in the format) for all TIMESTAMP_WITH_TIME_ZONE columns including these pre-filled values. It gets around that by specifying the time zone as `hiveStorageTimeZone` https://github.com/prestosql/presto/blob/master/presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java#L368